android/beagle/src/main/java/br/com/zup/beagle/android/view/BeagleActivity.kt

```diff
@@ -41,10 +42,35 @@ import br.com.zup.beagle.core.ServerDrivenComponent
 import kotlinx.android.parcel.Parcelize
 
 sealed class ServerDrivenState {
-    open class Error(val throwable: Throwable, val retry: BeagleRetry) : ServerDrivenState()
+
     class FormError(throwable: Throwable, retry: BeagleRetry) : Error(throwable, retry)
     class WebViewError(throwable: Throwable, retry: BeagleRetry) : Error(throwable, retry)
+    @Deprecated(DEPRECATED_STATE_LOADING)
     data class Loading(val loading: Boolean) : ServerDrivenState()
+
+    /**
+     * indicates that a server-driven component fetch has begun
+     */
+    open class Started: ServerDrivenState()
```

Comment:
there is some reason to use `open class`? what do you think to convert to `object`
```suggestion
    object Started: ServerDrivenState()
```

Reply:
done